### ISSUE TITLE
Fix jsx-a11y/alt-text warnings raised by jsx-a11y linter.

### ISSUE THREAD

**Issue by yzen** [devtools-html/debugger.html#7539]
To get a list of accessibility warnings raised by jsx-a11y linter run: `yarn lint:jsx-a11y`.

From that list please note all instances that are flagged to have "jsx-a11y/alt-text" problem. Each one needs to be addressed individually.

Once all of the instances are resolved make sure to change this setting in `.eslintrc.jsx-a11y` from:
```
"jsx-a11y/alt-text": "warn",
```

to
```
"jsx-a11y/alt-text": "error",
```

**Comment by Garbee** [comment]
This error is the golden old, no alternate text to image or object content for screen readers to fallback to. There was some talk today on the call of changing the images to spans due to the new border box for empty src attributes. That could make the immediate error here irrelevant directly. However, we should still make sure the new span method provides an accessible fallback for the content.
